src/components/CustomImage/CustomImage.tsx

```diff
@@ -0,0 +1,20 @@
+'use client';
+
+import { useState } from 'react';
+import Image, { ImageProps } from 'next/image';
+import type { StaticImageData, StaticRequire } from 'next/dist/shared/lib/get-img-props';
+import { errorImg } from '@/public/index';
+
+interface CustomImageProps extends ImageProps {
+  errorSrc?: string | StaticRequire | StaticImageData;
+}
+
+export default function CustomImage(props: CustomImageProps) {
+  const { alt, src, errorSrc, ...imageProps } = props;
+  const [imgSrc, setImgSrc] = useState(src);
+
+  const handleImageError = () => {
+    setImgSrc(errorSrc ?? errorImg);
+  };
+  return <Image alt={alt} src={imgSrc} {...imageProps} onError={handleImageError} />;
```

Comment:
```suggestion
  return <Image src={imgSrc} {...imageProps} onError={handleImageError} />;
```
alt도 { ...imageProps } 들어가지 않을까요? 근데 실수방지라면 이해완입니다. 저도 가져다 쓸래여 ㅎ

Reply:
왜 그런지 모르겠는데, 안 넣으면 alt를 넣어야 한다고 오류 뜨네요...
ImageProps에 alt가 옵셔널이 아닌데도 오류가 발생하다러구요...;